src/@next/Banner/Banner.tsx

```diff
@@ -53,43 +64,82 @@ export const Banner = React.forwardRef<HTMLDivElement, BannerProps>(
       ? iconName
       : iconByStatus || iconNameStatusMap['info'];
 
+    const actionComponent = (
+      <>
+        {action && (
```

Comment:
what if I only define secondary action but not action? 🤔

Reply:
no buttons will be rendered for now

Reply:
because I think you're not supposed to use secondaryAction if you don't give action in the first place, or wait a minute I have an alternative:

Reply:
@michac789 I don't think there's a validation logic stopping users from doing that?

Reply:
```
      <>
        {(action || secondaryAction) && (
          <ButtonGroup>
            {action && action}
            {secondaryAction && secondaryAction}
          </ButtonGroup>
        )}
      </>
```

Reply:
I think can just let action  & secondary action render without condition 
   ```
   <>
        {(action || secondaryAction) && (
          <ButtonGroup>
            { action}
            {secondaryAction}
          </ButtonGroup>
        )}
      </>
```

Reply:
IIRC undefined and null will be rendered as nothing with react, can verify this with the visual tests

Reply:
yup" you're right I just checked it... I wasn't really aware of this before, thanks